core/startos/src/service/service_effect_handler.rs

```diff
@@ -475,7 +461,7 @@ async fn get_primary_url(
 async fn list_service_interfaces(
     context: EffectContext,
     data: ListServiceInterfacesParams,
-) -> Result<BTreeMap<ServiceInterfaceId, ServiceInterfaceWithHostInfo>, Error> {
+) -> Result<BTreeMap<ServiceInterfaceId, ServiceInterface>, Error> {
```

Comment:
```suggestion
) -> Result<BTreeMap<ServiceInterfaceId, ServiceInterfaceWithHostInfo>, Error> {
```

this still needs to return `ServiceInterfaceWithHostInfo`. readd that type, and interpolate the host info from the host id in the body of this function